client/src/views/fields/base.js

```diff
@@ -269,9 +269,12 @@ Espo.define('views/fields/base', 'view', function (Dep) {
                 let tooltipTextTranslate = null;
                 if (this.getMetadata().get(['entityDefs', this.model.urlRoot, 'fields', this.name])) {
```

Comment:
// i think my code is better. Do you agree?
let tooltipText = this.getMetadata().get(['entityDefs', this.model.urlRoot, 'fields', this.name, 'tooltipText']);
if (tooltipText) {
   tooltipTextTranslate = this.translate(tooltipText, 'tooltips', this.model.name);
}